dbinit/src/main/liquibase/updates/changelog-0004-yd-417.yml

```diff
@@ -0,0 +1,23 @@
+databaseChangeLog:
+- changeSet:
+    id: 1490127260476-1
+    author: avdijk (generated)
+    changes:
+    - addColumn:
+        columns:
+        - column:
+            name: last_status_change_time
+            type: varchar(255)
+        tableName: buddies
+- changeSet:
+    id: 1491330640527-1
+    author: avdijk (generated)
+    changes:
+    - addColumn:
+        columns:
+        - column:
+            constraints:
+              nullable: false
```

Comment:
OK sorry. The default value indeed is `0` for existing records. I will add the default here explicitly indeed.